doc/source/user_guide/reshaping.rst

```diff
@@ -726,6 +726,92 @@ To choose another dtype, use the ``dtype`` argument:
 
     pd.get_dummies(df, dtype=bool).dtypes
 
+.. versionadded:: 1.4.0
+
+To convert a "dummy" or "indicator" ``DataFrame``, into a categorical ``DataFrame``,
+for example ``k`` columns of a ``DataFrame`` containing 1s and 0s can derive a
+``DataFrame`` which has ``k`` distinct values
```

Comment:
I can't figure out how to read this sentence, is part of it missing?

Reply:
Yep, I forgot a "using"